### ISSUE TITLE
Nanotrasen Hiring Standards Need to be changed [Blob Bug]

### ISSUE THREAD

**Issue by Jarektheraptor** [tgstation/tgstation#31940]
As a CMO, I was killed during a revolution, and a blob overmind RNG event showed up, which I managed to roll.

Due to a bug I've also reported I was unable to actually become the blob, and so the overmind body was given away to someone else, who proceeded to reach critical mass and kill everyone, winning the revolution for the heads.

This is a problem because the blob overmind was considered to be me, the CMO, and counted towards me being alive and surviving as the blob overmind. Here's a reference picture:

https://gyazo.com/c009618fcdfa3dcd22d7056e5e6cc8b0

Tl;dr Head of staff blob cucked the revolution. Promote blob when

**Comment by RogueSteampunker** [comment]
I see nothing wrong here. It's just the CMO releasing all of his hatred of being mistreated into become a cancerous growth onto the station.

#EqualBlobberRights

**Comment by KorPhaeron** [comment]
@AnturK 

This is probably my fault but pinging you anyway in case its a datum rev thing

**Comment by bgobandit** [comment]
lizards are already cancerous growths on the station so this is just you assuming your true form

**Comment by Anonmare** [comment]
Evidently a Feature. This is a buff to CMO during rev clearly
